api.swagger.yaml

```diff
@@ -12268,7 +12268,34 @@ components:
               type: boolean
               example: false
               readOnly: true
+            free_skills:
+              description: Free Skills
+              type: array
+              items:
+                $ref: '#/components/schemas/free-skill-schema'
+            personal_interests:
+              description: Personal Interests
+              type: array
+              items:
+                $ref: '#/components/schemas/personal-interest-schema'
+            social_media:
```

Comment:
social_media should be in user-schema-common (rule is if its same in GET & POST should be in common)